test/units/EverlongPortfolio.t.sol

```diff
@@ -119,4 +122,138 @@ contract TestEverlongPositions is EverlongTest {
             "position count should be 0 after opening and closing a long for the full bond amount"
         );
     }
+
+    /// @dev Validate that `hasMaturedPositions()` returns false
+    ///      with no positions.
+    function test_hasMaturedPositions_false_when_no_positions() external view {
+        // Check that `hasMaturedPositions()` returns false
+        // when no positions are held.
+        assertFalse(
+            everlong.hasMaturedPositions(),
+            "should return false when no positions"
+        );
+    }
+
+    /// @dev Validate that `hasMaturedPositions()` returns false
+    ///      with no mature positions.
+    function test_hasMaturedPositions_false_when_no_mature_positions()
+        external
+    {
+        // Open an unmature position.
+        everlong.exposed_handleOpenPosition(block.timestamp + 1, 5);
+
+        // Check that `hasMaturedPositions()` returns false.
+        assertFalse(
+            everlong.hasMaturedPositions(),
+            "should return false when position is newly created"
+        );
+    }
+
+    /// @dev Validate that `hasMaturedPositions()` returns true
+    ///      with a mature position.
+    function test_hasMaturedPositions_true_when_single_matured_position()
+        external
+    {
+        // Open unmatured positions with different maturity times.
+        everlong.exposed_handleOpenPosition(2, 5);
+        everlong.exposed_handleOpenPosition(3, 5);
```

Comment:
This is confusing to me. We start in the test harness by advancing time by 2 full position durations. I'd think that a maturity time of 2 or 3 would be far in the past. What am I missing?

Reply:
old test that was originally using just the portfolio struct and not everlong, didn't update it properly